x-pack/plugins/cases/server/client/attachments/get.ts

```diff
@@ -90,26 +85,15 @@ export async function find(
   { caseID, queryParams }: FindArgs,
   clientArgs: CasesClientArgs
 ): Promise<CommentsResponse> {
-  const {
-    unsecuredSavedObjectsClient: soClient,
-    caseService,
-    logger,
-    authorization,
-    auditLogger,
-  } = clientArgs;
+  const { unsecuredSavedObjectsClient: soClient, caseService, logger, authorization } = clientArgs;
```

Comment:
I understand it's not part of this PR, and we can do this in a followup, but I'd really like to see the `unsecuredSavedObjectsClient` name propogated everywhere, instead of renaming it to `soClient` or similar. The rename hides the fact that we are dealing with an unsecured SO client, which is dangerous IMO

Reply:
Sounds good! I added it as a line item here: https://github.com/elastic/kibana/issues/100804 we'll make sure it's completed in the RBAC PR as well.